src/components/ArticleList.jsx

```diff
@@ -2,20 +2,21 @@ import { useEffect, useState } from "react";
 import fetchArticles from "../controllers/fetchArticles";
 import ArticleListCard from "./ArticleListCard";
 import { Stack } from "@mui/material";
```

Comment:
As you're already using MUI in this project you could explore using icons, a 'Home' icon particularly would be a useful addition